hack/update-gomod.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
shall we always use this tool when update-gomod?

Reply:
No. Just use it to upgrade Kubernetes.